src/transformers/file_utils.py

```diff
@@ -956,6 +956,39 @@ def filename_to_url(filename, cache_dir=None):
     return url, etag
 
 
+def get_cached_models(cache_dir: Union[str, Path] = None) -> List[Tuple]:
+    """
+    Returns a list of tuples representing model binaries that are cached locally. Each tuple has shape `(model_url,
+    etag, size_MB)`. Filenames in `cache_dir` are use to get the metadata for each model, only urls ending with `.bin`
+    are added.
+
+    Args:
+        cache_dir (Union[str, Path], optional): Specify a cache directory to search for models within. Defaults to None.
+
+    Returns:
+        List[Tuple]: List of tuples each with shape `(model_url, etag, size_MB)`
+    """
+    if cache_dir is None:
+        cache_dir = TRANSFORMERS_CACHE
+
+    if isinstance(cache_dir, Path):
```

Comment:
```suggestion
        cache_dir = TRANSFORMERS_CACHE
    elif isinstance(cache_dir, Path):
```
Small nit, this can be regrouped like this.